### DISCUSSION TITLE
Do I need to download the streams?

### DISCUSSION THREAD

**Discussion by Mordizerr** [cyberofficial/Synthalingua#76] [Q&A]
Hello! I came across this software from your reddit post, and tried to install it. 
I think I followed every step correctly, but when I try to run the program, this is what is given to me, before closing on its own. 
What could be the problem? The hungarian text translates to after WinError2: The system didn't manage to find the given file
![WindowsTerminal_MomPu8BvV8](https://github.com/cyberofficial/Synthalingua/assets/65012367/d6e01b9b-77f1-4fd6-9841-285f5b751111)

And the Wrapper looks like this when I run it:
![Synthalingua_Wrapper_tstgO1pE6P](https://github.com/cyberofficial/Synthalingua/assets/65012367/cbd36e3e-17fc-49f9-b128-7d94bebaff27)

**Comment by cyberofficial** [comment]
Hello! Please follow this step in the troubleshooting section of the readme: https://github.com/cyberofficial/Synthalingua#:~:text=%5BWinError%202%5D%20The%20system%20cannot%20find%20the%20file%20specified%20Try%20this%20fix%3A%20%232%20(comment)

* [WinError 2] The system cannot find the file specified Try this fix: https://github.com/cyberofficial/Synthalingua/issues/2#issuecomment-1491098222

**Comment by Mordizerr** [comment]
Thank you for the fast answer, but that didn't do it for me. It still says that the file couldn't be found. Here are the things that were required on that guide:
![rundll32_kh5Wa0UOMy](https://github.com/cyberofficial/Synthalingua/assets/65012367/6a5addbe-63b7-434f-a929-126799d1c205)
![explorer_WSFws7oBFk](https://github.com/cyberofficial/Synthalingua/assets/65012367/c555051c-bcad-49d1-b90c-30146f7637fd)

And it still does this:
![7j01PmETB5](https://github.com/cyberofficial/Synthalingua/assets/65012367/1a67b5f6-a765-4cda-bd13-a70f5dc11591)

**Comment by cyberofficial** [comment]
I think I know the issue now, I should have seen earlier. As of right now, the program can only do **Live** video streams, meaning older broadcasts will not work.

This past stream i see you are using https://www.youtube.com/watch?v=9G-XZukRQzM will not work for live translating.

I would suggest you try using an fed audio input instead , you can check out this thread https://github.com/cyberofficial/Synthalingua/discussions/12#discussioncomment-5499327 

Requires the installation of virtual audio cable (like voicemeeter https://voicemeeter.com/) or something equivalent to pass audio to the program.


Currently under work i'm making an easier way to convert older past videos into videos with captions for easier viewing.

**Comment by cyberofficial** [comment]
Adding onto my previous comment 
Update 715cedc983123a93d66fd2b22f932b31aea94da2
https://github.com/cyberofficial/Synthalingua/blob/bleeding-under-work/modules/sub_gen.py

I'm currently working on said feature to automate older stream downloading and generate captions for said streams, so it'll be part of the next release soon, again sorry for the inconvenience. I'm just solo dev working on this project so cool features and additions takes time.

**Comment by Mordizerr** [comment]
Oh, got it. I didn't know there was a difference between live and older streams. I will try it out with a microphone and get back to you. Thank you so far very much!
